user/views.py

```diff
@@ -43,7 +43,7 @@ def privacy():
         user = User.objects.get(id=current_user.id)
         return render_template('user/privacy.html',
                            user=user,
-                           locations=user.locations)
+                           locations=sorted(user.locations, key=lambda l: l.date, reverse=True))
```

Comment:
Locations are sorted by default on save.
I guess `locations=user.locations.reverse()` should be enough.

Reply:
Oh sweet, way more efficient ! Thanks 🎉